internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -49,6 +52,21 @@ var Tables = []any{
 	&Trigger{},
 }
 
+func unmarshalData(dst any, j datatypes.JSON, c []byte) error {
```

Comment:
Same comment on API. Get data and boolean flag weather it's compressed. In general you can even skip this flag and look at first few bytes of data (magic, which for gzip is `1F8B`).